docs/home/export-formats.mdx

```diff
@@ -10,7 +10,9 @@ We currently export to the following file types: `.js`, `.ts`, `.css`, `.scss`,
 If you are using Mirrorful's web editor, you will need to create the imported files yourself. You can do this by simply creating a file of your desired format (i.e. `.js`, `.css`, etc) and then copying
 the generated code into the file.
 
-## Using CSS
+## Colors
```

Comment:
Great work, I'm thinking maybe Colors should be it's own page... instead of export formats being this massive file. We use Mintifly to set up docs btw: https://mintlify.com/docs/introduction

No worries if you don't want to make it's own page, just an idea.